pyvisa-py/gpib.py

```diff
@@ -507,7 +523,7 @@ class GPIBInterface(_GPIBCommon, Session):
 
     @staticmethod
     def list_resources():
-        return ['GPIB0::%d::INTFC' % pad for pad in _find_listeners()]
+        return ['GPIB%d::INTFC' % board for board, pad, in _find_boards()]
```

Comment:
The last comma in `board, pad,` is useless and should be removed for style uniformity.